### DISCUSSION TITLE
Feature request NCIRF

### DISCUSSION THREAD

**Discussion by choonsiklee** [ncidose/ncidosetools#36] [NCIRF]
Please leave new feature requests here.

**Comment by choonsiklee** [comment]
Anonymous

- Generate MCNP input files from Batch Manager
- Custom x-ray energy spectrum
- Implement pregnant phantoms
- Cone beam CT simulation

**Reply by choonsiklee** [reply]
NCIRF 2.0.20230506: Users can generate MCNPX/6 input files from Batch Manager for running on parallel computing servers.

**Comment by choonsiklee** [comment]
running batch configuration files on LINUX-based high performance computing servers

**Reply by ncidoseteam** [reply]
Included in NCIRF4.20260510

**Comment by choonsiklee** [comment]
PSA to +/-90 degree

**Reply by ncidoseteam** [reply]
Included in NCIRF4.20260510

**Comment by sstreitmatter** [comment]
Addition of mammography spectra for W/Rh, W/Al, W/Cu.

**Reply by ncidoseteam** [reply]
We plan to implement a feature for users to input custom x-ray spectrum in the next release.
